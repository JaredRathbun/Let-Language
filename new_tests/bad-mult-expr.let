let a, b := 2, 3, 4 in a + b + c