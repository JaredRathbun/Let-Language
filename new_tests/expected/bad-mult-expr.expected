Error (1): ID list and expr list must be of same length.
Parse failed.